src/Data/Connection/AbstractConnectionResolver.php

```diff
@@ -635,35 +709,18 @@ protected function execute_and_get_data() {
 		 */
 		$this->query = apply_filters( 'graphql_connection_query', $this->get_query(), $this );
 
-		/**
-		 * The items returned from the query. This array of items will be passed
-		 * to `get_nodes`
-		 *
-		 * Filter the items.
-		 *
-		 * @param array                      $items The items returned from the query
-		 * @param AbstractConnectionResolver $this  Instance of the Connection Resolver
-		 */
-		$items       = ! empty( $this->get_items() ) ? $this->get_items() : [];
-		$this->items = apply_filters( 'graphql_connection_items', $items, $this );
+		$this->ids = apply_filters( 'graphql_connection_ids', $this->get_ids(), $this );
```

Comment:
This hook is pending documentation per: https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-hooks-actions-and-filters